### PR TITLE
Fix build when wlroots is built without x11-backend

### PR DESCRIPTION
**Pull request by soreau**

### CODE REVIEW

File: src/core/output-layout.cpp [hunk 1]
```diff
@@ -240,7 +242,7 @@ namespace wf
             position_opt  = output_section->get_option("layout", default_value);
             scale_opt     = output_section->get_option("scale", "1");
             transform_opt = output_section->get_option("transform", "normal");
-
+#if WLR_HAS_X11_BACKEND
```
Comment (ammen99) [inline src/core/output-layout.cpp]: This breaks resizing on wayland backend, if x11 backend is disabled.

Comment (soreau) [reply src/core/output-layout.cpp]: @ammen99 I pushed a new version that should make more sense.

Comment (ammen99) [reply src/core/output-layout.cpp]: I'd prefer to make it without an additional function. Maybe something along the lines of

`bool is_nested_compositor = is_wyaland..()` and then inside an ifdef `is_nested... |= is_xwl()`

Comment (soreau) [reply src/core/output-layout.cpp]: You're right, that's cleaner.

**Comment by ammen99** [comment]
Looks good now, thanks!

**Pull request merged**